src/main/java/org/springframework/data/couchbase/core/query/Query.java

```diff
@@ -93,6 +120,28 @@ public Query with(final Sort sort) {
 		return this;
 	}
 
+	/**
+	 * inlineN1qlQuery (Query Annotation)
+	 */
+	public Query setInlineN1qlQuery(String inlineN1qlQuery) {
```

Comment:
If we construct it from Query.raw(), we don't need this, right?

Reply:
Right. The string would be an argument to the constructor.